packages/paste-website/src/pages/blog/2022-04-06-paste-newsletter.mdx

```diff
@@ -0,0 +1,235 @@
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    overview2: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/articles/pastes-path-to-a-transparent-package-categorization-system/overview-2.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
+
+<ArticleHeader
+  title={props.data.mdx.frontmatter.title}
+  description={props.data.mdx.frontmatter.description}
+  date={props.data.mdx.frontmatter.date}
+  machineDate={props.data.mdx.frontmatter.machineDate}
+  author={props.data.mdx.frontmatter.author}
+  avatar={props.data.mdx.frontmatter.avatar}
+  authorLink={props.data.mdx.frontmatter.author_link}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} hideFeedback />
+
+<ArticleContent>
+
+## 🚀 What's new in Paste
+
+### Paste goes international!
+
+![An example of Paste's form-pill-group component in French](../../assets/images/articles/2022-04-06-paste-newsletter/i18n-pill-group.png)
+
+To better support Twilio’s global efforts, Paste now supports all P1 languages. This includes a mechanism to swap the font for non-latin character sets, such as Japanese, to get a really crisp UI.
+
+There are some components that need a bit of extra attention when being used for
+multiple languages (for example, [Anchor requires an additional prop for i18n](/components/anchor/#internationalization)).
+Look for [an "internationalization" example](/components/button/#internationalization) on component docs pages to know whether a given component needs any updates in order to be translated into other languages.
+
+### 🌚 Paste Dark theme is ready for you!
+
+![An screenshot of the dark theme toggle on the Paste website, with dark theme selected](../../assets/images/articles/2022-04-06-paste-newsletter/dark-theme-switcher.png)
+
+We are excited to share that the Dark theme is coming out of early preview and is ready for general availability! We incorporated your feedback (buttons are fixed, y’all!), and we brought the Dark theme up to our accessibility and inclusive design standards. The Dark theme improvements are now live on the Paste site, so head on over and switch the toggle in the topmast to “Dark” to check it out.
+
+<Paragraph>
+  <Button as="a" href="/" variant="primary">
+    Check out the Dark Theme
+  </Button>
+</Paragraph>
+
+### The Badge component enters Stage 2
+
+![The new and improved Badge component in Paste](../../assets/images/articles/2022-04-06-paste-newsletter/badge-2.png)
+A big [Badge component](/components/badge/) update improves the look in both the default and dark themes, as well as improving the interactivity for clickable Badges. We’ve also introduced “decorative” variants that can be used for attributes that don’t have a semantic meaning but would still benefit from the visual hierarchy of a Badge.
```

Comment:
Suggestion: there's inconsistent spacing for the images and the paragraphs after; I added some spaces to the ones that need it!

```suggestion
![The new and improved Badge component in Paste](../../assets/images/articles/2022-04-06-paste-newsletter/badge-2.png)

A big [Badge component](/components/badge/) update improves the look in both the default and dark themes, as well as improving the interactivity for clickable Badges. We’ve also introduced “decorative” variants that can be used for attributes that don’t have a semantic meaning but would still benefit from the visual hierarchy of a Badge.
```